rosa_install_access_delete_clusters/terraform/rosa-sts-creating-a-cluster-with-customizations-terraform.adoc

```diff
@@ -0,0 +1,68 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="rosa-sts-creating-a-cluster-with-customizations-terraform"]
+= Customizing a ROSA cluster with Terraform
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-sts-creating-a-cluster-with-customizations-terraform
+
+toc::[]
+
+You can create a {product-title} (ROSA) cluster by using Terraform to customize the cluster's details.
+
+This guide assumes an understanding of ROSA. See the additional resources for conceptual information about {product-title} clusters.
+
+.Prerequisites
+
+* You have generated and stored an link:https://console.redhat.com/openshift/token[offline {cluster-manager-first} token].
+* You have an AWS account with the correct permissions.
+* You have installed the `terraform` CLI tool.
+
+[discrete]
+[id="further_reading_{context}"]
+=== Further reading
+
+* xref:../../rosa_planning/rosa-cloud-expert-prereq-checklist.adoc[Prerequisites checklist for deploying ROSA using STS]
+* xref:../../rosa_planning/rosa-understanding-terraform.adoc[Preparing Terraform to install ROSA clusters]
+
+[id="rosa-sts-creating-a-cluster-with-customizations-terraform-creating"]
+== Creating a {product-title} cluster using Terraform
+The following Terraform cluster example shows how to create your account-wide IAM roles and a ROSA cluster with a managed OIDC configuration.
+
+include::modules/rosa-sts-cluster-terraform-setup.adoc[leveloffset=+2]
+include::modules/rosa-sts-cluster-terraform-file-creation.adoc[leveloffset=+2]
+include::modules/rosa-sts-cluster-terraform-execute.adoc[leveloffset=+2]
+include::modules/rosa-sts-cluster-terraform-destroy.adoc[leveloffset=+2]
+
+[id="rosa-sts-creating-a-cluster-with-customizations-terraform-customizing"]
+== Terraform customization options
+
+The following sections detail individual customizations you can add to your `main.tf` and `terraform.tfvar` files.
+
+include::modules/rosa-cluster-cluster-role-name-change.adoc[leveloffset=+2]
+
+[id="next-steps_{context}"]
+== Next steps
+
+* xref:../../rosa_install_access_delete_clusters/rosa-sts-accessing-cluster.adoc#rosa-sts-accessing-cluster[Accessing a ROSA cluster]
+
+[role="_additional-resources"]
+[id="additional-resources_rosa-sts-creating-a-cluster-with-customizations-terraform"]
+== Additional resources
+
```

Comment:
I like how this section is done in the current docs for non-Terraform clusters, where each bullet is a sentence that says *why* the user might go to said additional resource. You may want to consider doing that here. Plus, parallelism. 
<img width="733" alt="Screenshot 2024-01-26 at 9 44 45 AM" src="https://github.com/openshift/openshift-docs/assets/119696027/50c58415-3c67-43ba-9f7c-ab07af46056e">